### ISSUE TITLE
How do I see code evaluation problems?

### ISSUE THREAD

**Issue by ilanc** [data-forge-notebook/wiki#8]
Hi Ashley,
Phenomenal product thanks!

How do I see evaluation errors? like `ReferenceError` etc... e.g. given the following notebook:
```js
let a = 10;
console.log(a);
console.log(a + b); // ReferenceError: b is not defined
```
Upon evaluation DFN just runs up to the ReferenceError and fails silently see pic

![image](https://user-images.githubusercontent.com/1793957/103639480-1780d400-4f57-11eb-9e11-46b8b160de33.png)

This makes it difficult to see where I've got simple .js syntax errors.

P.S. I'm using 1.10.2 on Ubuntu 18

**Comment by ashleydavis** [comment]
That's very strange! The error should be printed after the code cell.

I'm looking into it.

**Comment by ashleydavis** [comment]
I've tested this in version 10.2 and there is no error message.

I then tested it in the dev version 10.3 (not yet released) and it does seem to be working:

![image](https://user-images.githubusercontent.com/625126/103733585-33ab7080-503e-11eb-87c2-d888189e954b.png)

**Comment by ashleydavis** [comment]
Very strange. I did a prod build and the error message doesn't show up in the prod version of 10.3.

So this must be an issue only in prod.

**Comment by ilanc** [comment]
I did get some errors, but not Ref errors and other syntax type errors. Are you invoking node with different flags or env vars in prod perhaps?

**Comment by ashleydavis** [comment]
I think I see the problem. It looks like a file required to evaluate source maps (and pin point the location of the error) is being copied to the wrong place in the prod build. Looks like it's been an issue since 1.10.0.

**Comment by ilanc** [comment]
While I have you on the line ... I have a few feature requests - should I email you directly? or is the issue tracker the best place?

**Comment by ashleydavis** [comment]
Feel free to email me on ashley@codecapers.com.au.

**Comment by ashleydavis** [comment]
This issue is fixed now. Thanks for your patience.

I'll include the fix in release 1.10.3 in the next couple of weeks. 

Please feel free to reopen this issue then if the problem reoccurs.
